addons/sale/models/sale_order.py

```diff
@@ -216,8 +216,8 @@ def _rec_names_search(self):
     amount_untaxed = fields.Monetary(string="Untaxed Amount", store=True, compute='_compute_amounts', tracking=5)
     amount_tax = fields.Monetary(string="Taxes", store=True, compute='_compute_amounts')
     amount_total = fields.Monetary(string="Total", store=True, compute='_compute_amounts', tracking=4)
-    amount_to_invoice = fields.Monetary(string="Amount to invoice", store=True, compute='_compute_amount_to_invoice')
-    amount_invoiced = fields.Monetary(string="Already invoiced", compute='_compute_amount_invoiced')
+    amount_to_invoice = fields.Monetary(string="Un-invoiced Balance", compute='_compute_invoicing_amounts', store=True)
+    amount_invoiced = fields.Monetary(string="Already invoiced", compute='_compute_invoicing_amounts', store=True)
```

Comment:
This needs an upgrade script. Do we actually need to store?

Reply:
No need to store them indeed :slightly_smiling_face: